delivery_task.rb

```diff
@@ -0,0 +1,53 @@
+require 'net/http'
+require 'pry'
+require 'json'
+
+class Delivery
+
+  def self.routs(origins, destinations)
+  	if origins == "" || destinations == ""
+  		raise "Origins or Destinations is empty"
+  	elsif
+  		origins == destinations
+  		raise "You can't enter two identical values"  		
```

Comment:
What happens if we call method with same city in different register or string with one space (" ")?